Snakefile

```diff
@@ -524,8 +524,10 @@ rule evaluation_ensemble_pr_curve:
         pr_curve_file = SEP.join([out_dir, '{dataset_gold_standard_pairs}-eval', 'pr-curve-ensemble-nodes.txt']),
     run:
         node_table = Evaluation.from_file(input.gold_standard_file).node_table
-        node_ensemble_dict = Evaluation.edge_frequency_node_ensemble(node_table, input.ensemble_file, input.dataset_file)
-        Evaluation.precision_recall_curve_node_ensemble(node_ensemble_dict, node_table, output.pr_curve_png, output.pr_curve_file)
+        input_interactome = Evaluation.from_file(input.dataset_file).get_interactome()
+        node_ensemble_dict = Evaluation.edge_frequency_node_ensemble(node_table, input.ensemble_file, input_interactome)
+        input_nodes = Evaluation.from_file(input.dataset_file).get_node_columns(["sources", "targets", "prize", "active"])
```

Comment:
Can you store the dataset the first time you get it `from_file`?

We may want a new helper function in Dataset to take the place of `get_node_columns(["sources", "targets", "prize", "active"])` so that isn't hard-coded here. Something that would say "get all of the columns that should be considered interesting input" so that if new algorithms update that in the future we don't have to track down all the places in the source that list those node properties.